modules/ROOT/pages/import-api-specification-exchange.adoc

```diff
@@ -55,8 +55,61 @@ Manage the API specifications linked to Exchange in your project from the API sp
 [NOTE]
 If you want Studio to list your API when searching in Exchange, you must mark your API specification published as an asset as *stable*.
 
+== Implement AsyncAPI Specifications
+
+Studio supports the implementation of AsyncAPI 2.6 specs. When creating an implementation project, Studio imports and scaffolds an AsyncAPI spec hosted on Exchange into an API interface that you can implement.
```

Comment:
do we also support 2.0?

Reply:
No, Studio does not support 2.0 spec. Thanks!